apps/subscriber-app-e2e/src/integration/subscription-management/subscription-management.step.ts

```diff
@@ -135,3 +135,31 @@ And('the user views contact information of {string}, {string} and {string}', fun
   }
   subscriptionManagementObj.preferredNotificationChannelDisplay().should('have.value', channel);
 });
+
+Then('the user views the checked {string} icon for {string}', function (channel, subscriptionName) {
+  switch (channel.toLowerCase()) {
+    case 'email':
+      subscriptionManagementObj
+        .availableChannel(subscriptionName)
+        .get('[data-testid="' + channel.slice(1, channel.length) + '-icon"]')
```

Comment:
In this case branch, channel variable equals "email", the get statement can be just be .get('[data-testid="mail-icon"]) to be simpler.